File: clear-all/clear-all/src/main/java/com/example/clear_all/dto/response/CategoriesQuery.java
package com.example.clear_all.dto.response;


import lombok.Data;

import java.math.BigDecimal;
import java.math.BigInteger;
import java.util.Date;

@Data
public class CategoriesQuery {
    private BigDecimal id;
    private String categoryName;
    private String slug;
    private BigInteger parentId;
    private BigInteger displayOrder;
    private BigInteger createBy;
    private Date createDate;
    private BigInteger lastUpdateBy;
    private Date lastUpdateDate;
}

File: WebSport24hNews/Application/Query/Model/24hCategories/CategoriesQuery.cs
using Microsoft.EntityFrameworkCore;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace WebSport24hNews.Application.Query.Model._24hCategories
{
    public class CategoriesQuery
    {
        public decimal Id { get; set; }
        public string? CategoryName { get; set; }
        public string? Slug { get; set; }
        public decimal? ParentId { get; set; }

        public decimal? DisplayOrder { get; set; }
        public decimal? LeagueId { get; set; }

        public decimal CreateBy { get; set; }

        public DateTime CreateDate { get; set; }

        public decimal? LastUpdateBy { get; set; }

        public DateTime LastUpdateDate { get; set; }
    }
}
